env.d.ts

```diff
@@ -710,10 +719,72 @@ type Nullsec = Lowsec & PlayerNullsec & {
 	}
 }
 
+const MongoTypes = {
```

Comment:
this is creating a global variable named `MongoTypes`. is there really a global variable named `MongoTypes` in the hackmud runtime?

Reply:
Yeah that doesn't exist in the hackmud runtime. I was wondering with the understanding that it wouldn't be included in the list of globals because of the lack of `declare` or `export`. Should I just rewrite this as a pure type rather than getting the type/keys based on the `MongoTypes`?